src/App.vue

```diff
@@ -1,39 +1,28 @@
 <template>
-  <div>
-    <nav>
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
-    </nav>
+  <DefaultLayout>
     <router-view />
-  </div>
+  </DefaultLayout>
 </template>
 
-<style>
+<script lang="ts">
+import DefaultLayout from "@/views/layout/Default.vue";
+
+export default {
+  components: {
+    DefaultLayout,
+  },
+};
+</script>
 
+<style>
 @tailwind base;
 
 @tailwind components;
 
 @tailwind utilities;
+@import url("https://fonts.googleapis.com/css?family=Poppins:300,400,500,600,700");
```

Comment:
Add new line before this import. Phần trên là blcok của tailwind, phần này import cái khác rồi. Với em nghiên cứu làm sao dùng yarn để cài poppin về dự án chứ k nên dùng CDN thế này đâu, trường hợp k/h không có internet thì font k nạp được